src/main/resources/spring/camel-context.xml

```diff
@@ -168,10 +200,10 @@
                     <choice>
                         <when>
                             <simple>${header.filterText} == null</simple>
-                            <bean ref="analysisService" method="findAllByOwner(${header.${type:org.jboss.xavier.integrations.route.RouteBuilderExceptionHandler.USERNAME}}, ${header.page}, ${header.size})" />
+                            <bean ref="analysisService" method="findAllByOwner(${header.${type:org.jboss.xavier.integrations.route.RouteBuilderExceptionHandler.USERNAME}}, ${header.offset}, ${header.limit})" />
                         </when>
                         <otherwise>
-                            <bean ref="analysisService" method="findByOwnerAndReportName(${header.${type:org.jboss.xavier.integrations.route.RouteBuilderExceptionHandler.USERNAME}}, ${header.filterText}, ${header.page}, ${header.size})" />
+                            <bean ref="analysisService" method="findByOwnerAndReportName(${header.${type:org.jboss.xavier.integrations.route.RouteBuilderExceptionHandler.USERNAME}}, ${header.filterText}, ${header.offset}, ${header.limit})" />
```

Comment:
Maybe we can reuse the code from master branch so
1. we add `<to uri="direct:to-pageBean"/>` before `<choice>`
2. Instead of sending `${header.offset}, ${header.limit}` we can send `{header.pageBean}`
3. Transform the final response to use the new Page Response using `<bean ref="apiService" method="mapToCustomPage(${exchange})"/>`